test/mocha/test.js

```diff
@@ -0,0 +1,8 @@
+const { multiply } = require("./calc");
+var assert = require('assert');
+
+describe(multiply.name, function () {
+  it('multiplies numbers correctly', function () {
+    assert.equal(multiply(2, 5), 10);
```

Comment:
I just realized it's missing from the jest tests too (feel free to go ahead and add it while you're at it), but this test suite should also probably include a failing test. (Note the test harness will probably need to be modified to allow for non-zero exit codes.)

Reply:
I'm adding a failing test including a new function with an intended bug in calc.js.
Should I remove `assert(result.status === 0 && result.error === undefined);` in `helpers.ts:runCommand`, or add a function like `runAppmapNodeWithoutAssertion`?

Reply:
Yeah, I'd remove the assert and instead add `expect` checks on the exit status.